### ISSUE TITLE
Adding Multiplexers.

### ISSUE THREAD

**Issue by jmclemo6** [joas8211/Logic.js#1]
I think this would be a good idea as the multiplexer is a pretty ubiquitous design in digital logic.

**Comment by joas8211** [comment]
Doesn't this require modifing the base functionality (I/O) to use "analog" signals instead of digital, or numeric values instead of boolean values?

It isn't a big deal this early on the development (not even drawing implemented).

**Comment by jmclemo6** [comment]
Not necessarily.  Even a 2-to-1 MUX that takes two boolean inputs along with a select bit and has one boolean output could be useful.

**Comment by joas8211** [comment]
This feature will be implemented as soon as I just get back to this project, which may be never.

I don't plan to end this project here, but usually it goes like that with my bigger personal projects. I'd love to continue this but I'm not able to currently.

The only differense this time is there's at least someone interested in the project.
